CONTRIBUTING.md

```diff
@@ -151,6 +152,125 @@ Validate authentication using the username `alice` with the password `alice`:
 $ curl -v -k -X POST -d "alice" http://localhost:3000/authn-ldap/test/cucumber/alice/authenticate
 ```
 
+
+
+#### Google Cloud Platform (GCP) Authentication
+
+To enable a host to log into Conjur using GCP identity token, run `start` with the `--authn-ldap` flag and pass a name of a running Google Compute Engine (GCE).
+
+****Prerequisites****
+
+- A Google Cloud Platform account. https://cloud.google.com/.
+- Access to a running Google Compute Engine instance. 
+
+****Obtaining the Instance Identity Token****
+
+An identity token is obtained from the Google's metadata server. Refer to Google Cloud [documentation](https://cloud.google.com/compute/docs/instances/verifying-instance-identity#request_signature) on obtaining the instance identity token.
+To access Google's metadata server you need to run a *curl* command inside a running GCE instance.
+The below snippet depicts a `curl` request to obtain an identity token:
+
+``` 
+curl -H "Metadata-Flavor: Google" \
+'http://metadata/computeMetadata/v1/instance/service-accounts/default/identity?audience=[AUDIENCE]&format=[FORMAT]'
+```
+Replace the following:
+- *AUDIENCE* - Conjur host id, in the following format: `conjur/[conjur-account-name]/host/[host-id]`. (Audience is the unique URI agreed upon by both the token sender and receiver and used for validation of the token)` `
+- *FORMAT* - Conjur expects an identity token in full format. Use the value: `full`. Format is the parameter that specifies whether or not the project and instance details are included in the payload.
+
+Your request might look similar to the following example:
+
+```
+curl -G -H "Metadata-Flavor: Google" \
+--data-urlencode "audience=conjur/myorg/host/gce-app/testapp" --data-urlencode "format=full" \ "http://metadata/computeMetadata/v1/instance/service-accounts/default/identity"
+```
+
+****Configure Google Cloud Authenticator****
+
+###### Before you begin
+
+Ensure you have the following information:
+
+| What you need           | Description                                                  |
+| :---------------------- | :----------------------------------------------------------- |
+| Instance name           | The name of the instance to which this token belongs. This name can be reused by several instances over time, so use the `instance_id` value to identify a unique instance ID. |
```

Comment:
as per our dicsussions earlier: add the GCP/GCE identifier to project/instance/etc